src/components/LayerUI.tsx

```diff
@@ -37,6 +38,8 @@ interface LayerUIProps {
   onLockToggle: () => void;
 }
 
+const TOP_BOTTOM_APP_MENU_HEIGHT = 236;
```

Comment:
Would it be possible/reasonable to measure this directly at runtime?

Reply:
I used ref to measure panel height at runtime. I referenced the guide in this [link](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node).